Examples/Swift/ViewController.swift

```diff
@@ -33,13 +33,19 @@ class ViewController: UIViewController, MGLMapViewDelegate {
 
     var exampleMode: ExampleMode?
     
+    var locManager = CLLocationManager()
```

Comment:
Nit: spell out `locationManager`.

Reply:
Looks like there’s a conflict with a preexisting `locationManager()`. Can we rename that method?